site/pages/sysadmin/org.page.tsx

```diff
@@ -117,6 +117,7 @@ const ManageOrgs = (props: PageState<Partial<ManageOrgProps>>): ReactElement =>
 
                 <input type="hidden" name="PK" value={pageState.inputs.PK} />
                 <input type="hidden" name="adminAreaCodes" value={pageState.inputs.adminAreaCodes ?? []} />
+                <input type="hidden" name="mode" value={JSON.stringify(pageState.inputs.mode)} />
```

Comment:
will mode always be defined otherwise this would fail?

Reply:
yes it will always be returned because the org will always have this info in db